### ISSUE TITLE
[EventRequest] codeunit 99000845 "Reservation Management" - AutoReserveOneLine - OnBeforeAutoReserveOneLine

### ISSUE THREAD

**Issue by TKapitan** [microsoft/ALAppExtensions#19103]
Please extend the Event OnBeforeAutoReserveOneLine with CalcReservEntry so you will be able to find information about the requested reservation... We need this parameter to know info such as item/location codes so we can skip reservations for specific combinations

```
IsHandled := false;
OnBeforeAutoReserveOneLine(CalcReservEntry, AvailabilityDate, IsHandled);
if IsHandled then
	exit;

[IntegrationEvent(false, false)]
local procedure OnBeforeAutoReserveOneLine(var CalcReservEntry: Record "Reservation Entry"; var AvailabilityDate: Date; var IsHandled: Boolean)
begin
end;
```

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 43974.

**Comment by TKapitan** [comment]
It's still not available even in 20.5.45456.45889?

**Comment by JesperSchulz** [comment]
This is fixed for 21.0:
![image](https://user-images.githubusercontent.com/19796701/189861120-b07ca6df-06b2-4644-aed4-3d70fe58555c.png)

Not entirely sure why this wasn't backported to 20.x, but your fix will release in a couple of weeks. Sorry about the inconvenience. We've had a few cases where the backport to the latest release branch failed. We're investigating.
